storage/indexer/src/db_indexer.rs

```diff
@@ -413,6 +482,22 @@ impl DBIndexer {
 
         assert_eq!(num_transactions, version - start_version);
 
+        if self.indexer_db.event_v2_translation_enabled() {
+            batch.put::<InternalIndexerMetadataSchema>(
+                &MetadataKey::EventV2TranslationVersion,
+                &MetadataValue::Version(version - 1),
```

Comment:
what does -1 mean?

Reply:
`version - 1` means the last processed version number. This is consistent to other entries in `InternalIndexerMetadataSchema`. They are all recording the last processed version number.